torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -337,8 +337,10 @@ std::shared_ptr<FutureMessage> TensorPipeAgent::send(
 
   std::shared_ptr<FutureMessage> futureResponseMessage =
       std::make_shared<FutureMessage>();
+
   requestMessage.setId(nextMessageID_++);
-  pendingResponseMessage[requestMessage.id()] = futureResponseMessage;
+  auto requestMessageId = requestMessage.id();
+  pendingResponseMessage[requestMessageId] = futureResponseMessage;
```

Comment:
Nit:
```
auto requestMessageId = nextMessageID_++;
requestMessage.setId(requestMessageId);
```
This may be more readable than first setting the id on the message then getting it back.